sdk-cpp/src/do_exceptions.cpp

```diff
@@ -1,5 +1,4 @@
-// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT License.
```

Comment:
Retain copyright. You'll also need to scope the #if when implementing std::error_code return type.